### PR TITLE
Update pt_BR messages for the RP-initiated logout

### LINKED ISSUE #13083: Missing translated messages in Portuguese
### Describe the bug

The new user approval page for logout has untranslated messages

### Version

18.0.2

### Expected behavior

_No response_

### Actual behavior

_No response_

### How to Reproduce?

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by weltonrodrigo**
The new user approval page for logout has untranslated messages.

Resolves #13083

### CODE REVIEW

**Review by trixpan — DISMISSED** [review]

**Comment by trixpan** [comment]
@abstractj

**Review by abstractj — COMMENTED** [review]
@weltonrodrigo I'm not picky about it, but what @trixpan suggested makes sense to me. Would you be able to update your PR?

**Comment by weltonrodrigo** [comment]
> @weltonrodrigo I'm not picky about it, but what @trixpan suggested makes sense to me. Would you be able to update your PR?

done.

File: themes/src/main/resources-community/theme/base/account/messages/messages_pt_BR.properties [hunk 1]
```diff
@@ -347,3 +347,10 @@ openshift.scope.user_info=Informa\u00e7\u00f5es do usu\u00e1rio
 openshift.scope.user_check-access=Informa\u00e7\u00f5es de acesso do usu\u00e1rio
 openshift.scope.user_full=Acesso Completo
 openshift.scope.list-projects=Listar projetos
+
+# new RP-initiated logout
+frontchannel-logout.title=Saindo
+frontchannel-logout.message=Voc\u00EA est\u00E1 saindo dos seguintes aplicativos
+logoutConfirmTitle=Saindo
+logoutConfirmHeader=Voc\u00EA realmente quer sair?
```
[the remark this answers is not in the document]
Comment (weltonrodrigo) [reply themes/src/main/resources-community/theme/base/account/messages/messages_pt_BR.properties]: accepted

**Review by abstractj — APPROVED** [review]

**Pull request merged**